.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: bash
+sudo: true
```

Comment:
No, we need sudo access see https://github.com/SUSE-Cloud/automation/blob/master/Makefile#L48-L50.